x-pack/plugins/maps/public/components/_index.scss

```diff
@@ -3,3 +3,19 @@
 @import './layer_panel/index';
 @import './widget_overlay/index';
 @import './toolbar_overlay/index';
+
+// The following code will only get compiled from this file
+
+// Custom SVG as background for zoom controls based off of EUI glyphs plusInCircleFilled and minusInCircleFilled
+// Also fixes dark mode
+.mapboxgl-ctrl-icon.mapboxgl-ctrl-zoom-in {
+  background-repeat: no-repeat;
+  background-image: url("data:image/svg+xml,%0A%3Csvg width='15px' height='15px' viewBox='0 0 15 15' version='1.1' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='#{hexToRGB($euiTextColor)}' d='M8,7 L8,3.5 C8,3.22385763 7.77614237,3 7.5,3 C7.22385763,3 7,3.22385763 7,3.5 L7,7 L3.5,7 C3.22385763,7 3,7.22385763 3,7.5 C3,7.77614237 3.22385763,8 3.5,8 L7,8 L7,11.5 C7,11.7761424 7.22385763,12 7.5,12 C7.77614237,12 8,11.7761424 8,11.5 L8,8 L11.5,8 C11.7761424,8 12,7.77614237 12,7.5 C12,7.22385763 11.7761424,7 11.5,7 L8,7 Z M7.5,15 C3.35786438,15 0,11.6421356 0,7.5 C0,3.35786438 3.35786438,0 7.5,0 C11.6421356,0 15,3.35786438 15,7.5 C15,11.6421356 11.6421356,15 7.5,15 Z' /%3E%3C/svg%3E");
```

Comment:
15px i assume so it centers? Surprised this worked in ie11.

Reply:
15px is the actual size of the icon
<img width="276" alt="Screen Shot 2019-05-21 at 19 55 34 PM" src="https://user-images.githubusercontent.com/549577/58137982-69e67300-7c02-11e9-9acb-ab0778c49a9a.png">

Why is it surprising to have worked in IE?